action.yml

```diff
@@ -24,7 +24,7 @@ runs:
     - name: Install Foundry
       uses: foundry-rs/foundry-toolchain@v1
       with:
-        version: nightly
+        version: stable
```

Comment:
can we probably stick to the `v0.3.0` instead to make it pinned?

https://github.com/foundry-rs/foundry-toolchain?tab=readme-ov-file#inputs